evennia/commands/cmdsethandler.py

```diff
@@ -351,6 +351,14 @@ def update(self, init_mode=False):
                     elif path:
                         cmdset = self._import_cmdset(path)
                         if cmdset:
+                            if cmdset.key == '_CMDSET_ERROR':
+                                # If a cmdset fails to load, check if we have a fallback path to use
+                                fallback_path = settings.CMDSET_FALLBACKS.get(path, None)
```

Comment:
The `cmdset.key = "_CMDSET_ERROR"` is important btw - that is caught higher up in the chain and used for reporting in-game. For reporting to the user, the `cmdset.errmessage` is echoed to the user whenever a `cmdset.key == "_CMDSET_ERROR` is seen. There is nothing stopping us from doing additional logging though - especially if we quietly swap out the cmdset as done here. 
@TehomCD  We do not send tracebacks to a channel, this is true. I could see your point in doing that. I guess that could be a further use of the mudinfo channel maybe (or make a new one `syslog` for just errors and anomalies). Relaying to a channel should in that case just be an optional kwarg (with a default you can change from settings) of the `logger.log_err` function and its siblings. That would be a separate PR though. For now, this should just get logged.